### PR TITLE
chore (events) January 2022, part 1

### PR DESCRIPTION
**Pull request by TinaNEs**
added new events

### CODE REVIEW

File: data/events.xml [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+<p>Join us as Andy Goldberg talks about two packages, http4k and Hoplite, as used in combination. We'll walk through some code and some actual deployment options to see it in action. Anyone who would like a preview is welcome to read https://cfnine.com/blog/deploying-http4k.<p>
+]]>
+         </description>
+    </event>
+
+    <event>
+        <lang>en</lang>
+        <startDate>2022-01-06</startDate>
+        <endDate>2022-01-06</endDate>
+        <location>San Diego, CA</location>
```
Comment (meilalina) [inline data/events.xml:23]: @TinaNEs Неправильная дата и локация

File: data/events.xml [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+]]>
+         </description>
+    </event>
+
+
+    <event>
+        <lang>en</lang>
+        <startDate>2022-01-06</startDate>
+        <endDate>2022-01-06</endDate>
+        <location>San Diego, CA</location>
```
Comment (meilalina) [inline data/events.xml:45]: @TinaNEs wrong date and location

File: data/events.xml [hunk 3]
```diff
@@ -1,5 +1,136 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <events>
+    <event>
+        <lang>en</lang>
+        <startDate>2022-02-06</startDate>
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
```
Comment (meilalina) [inline data/events.xml:61]: @TinaNEs Fosdem is fully online this year. Please remove the location and add the "online" tag

File: data/events.xml [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+        <subject>Kotlin APIs with Flexible Deployment Options</subject>
+        <url>https://www.meetup.com/sd-kotlin/events/mjfhnsydccbhb/</url>
+        <description>
+            <![CDATA[
+<p>Join us as Andy Goldberg talks about two packages, http4k and Hoplite, as used in combination. We'll walk through some code and some actual deployment options to see it in action. Anyone who would like a preview is welcome to read https://cfnine.com/blog/deploying-http4k.<p>
+]]>
+         </description>
+    </event>
+
+    <event>
```
Comment (meilalina) [inline data/events.xml:19]: @TinaNEs it duplicates event from the previous commit https://github.com/JetBrains/kotlin-web-site/pull/2757/commits/55c2ac3b3c943c3949f3ca783d1813952580885a

File: data/events.xml [hunk 5]
```diff
@@ -1,5 +1,226 @@
...
+   </event>
+
+   <event>
+        <lang>en</lang>
+        <startDate>2022-02-06</startDate>
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
+        <speaker>Romain Boisselle</speaker>
+        <title>Fosdem 2022</title>
+        <subject>The beauty of Kotlin typing system</subject>
```
Comment (meilalina) [inline data/events.xml:306]: @TinaNEs seems the name is wrong

File: data/events.xml [hunk 6]
```diff
@@ -1,5 +1,226 @@
...
+]]>
+       </description>
+    </event>
+
+    <event>
+        <lang>en</lang>
+        <startDate>2022-02-06</startDate>
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
+        <speaker>Anton Arhipov</speaker>
```
Comment (meilalina) [inline data/events.xml:288]: @TinaNEs the speaker is wrong

File: data/events.xml (hunk 1 above)
Comment (TinaNEs) [reply data/events.xml:23]: added changes

File: data/events.xml (hunk 6 above)
Comment (TinaNEs) [reply data/events.xml:288]: name is already corrected

File: data/events.xml (hunk 5 above)
Comment (TinaNEs) [reply data/events.xml:306]: i made a mistake in the title. 
Now it is fixed.

**Pull request merged**